src/Handler/MicrosoftTeamsRecord.php

```diff
@@ -216,13 +216,16 @@ public function setSections(array $record): self
     }
 
     /**
-     * @param string $name
+     * @param mixed $name
```

Comment:
Please do not use `mixed`, but write all types you want to support like the interface `Stringable`. Arrays and objects without the `__toString` function are not allowed by the `strval` function.

Reply:
Sure. Now only `int` and `string` types are allowed.